web/cgi-bin/missa/propers.pl

```diff
@@ -82,7 +82,9 @@ sub specials {
     if ($item =~ $section_regex) {
       $label = $1;
 
-      if ($rule =~ /omit.*\b$label\b/i) {
+      if ( ($rule =~ /omit.*\b$label\b/i) ||
+           ( ( $version =~ /1570/) && ( $item =~ / Le/ ) ) # omit Leonine prayers issue #367
```

Comment:
A couple of things:

 - Identifying the Leonine prayers by this regex is far from ideal.  Better would be to arrange for the title of the section to be the same in all translations, so that it can be identified robustly, and then translate it for display as is done with the other section headers.
 - There are superfluous parentheses and whitespace here.

I'll merge this change as it is for now, though, as the code is already full of such things, but it would be best in future to try to make things more robust.